isbn/lib.php

```diff
@@ -69,7 +69,8 @@
     'produktSigel' => '//feld[@nr="078" and @ind="e"]/uf[@code="a"]',
 );
 
-function performMapping($map, $outputXml) {
+function performMapping($map, $outputXml)
+{
```

Comment:
Are the parantheses in functions and loops handled differently in this coding style?